build.gradle

```diff
@@ -43,10 +43,11 @@ ext {
 }
 
 repositories {
-     maven { url "https://m2.neo4j.org/content/repositories/snapshots" }
-     mavenCentral()
-     maven { url "https://m2.neo4j.org/content/repositories/snapshots" }
-     maven { url "http://oss.sonatype.org/content/repositories/snapshots/" }
+    maven { url "https://m2.neo4j.org/content/repositories/snapshots" }
+    mavenCentral()
+    maven { url "https://m2.neo4j.org/content/repositories/snapshots" }
+    maven { url "http://oss.sonatype.org/content/repositories/snapshots/" }
+    flatDir { dirs 'lib' }
```

Comment:
How will that run on other machines (also Travis?)

Reply:
in the "lib" folder there is the Oracle jar for tests.
we put it there for convenience, otherwise we can remove it but if someone starts the local test with oracle it must have it in his pc.